setup.cfg

```diff
@@ -1,20 +1,22 @@
 [metadata]
 name = infrasim
-summary = EMC GHTS infrasim
+summary = Infrasim Emulation for legacy Hardware servers
```

Comment:
Emulation vs Simulation  = to be or not to be :-) - prefer simulation
Suggested to remove legacy. Guess not everyone treat BMC based server as legacy server